backend/src/Designer/wwwroot/designer/css/startpage.css

```diff
@@ -365,6 +363,11 @@ img {
     background-position: left 120px;
 }
 
+.jumbotron.a-jumbotron-header {
```

Comment:
Not part of your PR, but why is `jumbotron` and `a-jumbotron` two different classes? 🤔  
From the [HTML](https://github.com/Altinn/altinn-studio/blob/670ce44b093167a4df038aafa74c2bbb956f00ea/backend/src/Designer/Views/Home/StartPage.cshtml), it looks like they can be merged together, making the code more readable.

Reply:
`jumbotron` comes from Bootstrap, while `a-jumbotron` comes from the first version of designsystem. The start page is an old page that needs to be cleaned up. I created an issue here: https://github.com/Altinn/altinn-studio/issues/13418."